cmd/pomerium/main.go

```diff
@@ -206,13 +209,22 @@ func parseOptions(configFile string) (*config.Options, error) {
 	if o.LogLevel != "" {
 		log.SetLevel(o.LogLevel)
 	}
+	metrics.AddPolicyCountCallback(o.Services, func() int64 {
+		return int64(len(o.Policies))
+	})
+	checksumInt, err := strconv.ParseInt(fmt.Sprintf("0x%s", o.Checksum()), 0, 64)
```

Comment:
Feels scary. But I suppose if this fails we get 0 not nil

Reply:
Yes both overflow and "not an int" conditions are safe based on my read of ParseInt() docs.